website/content/en/docs/kubebuilder/advanced-topics.md

```diff
@@ -60,14 +59,15 @@ import (
   routev1 "github.com/openshift/api/route/v1"
 )
 
-func main() {
+func init() {
   ...
 
   // Adding the routev1
-  if err := routev1.AddToScheme(mgr.GetScheme()); err != nil {
-    log.Error(err, "")
-    os.Exit(1)
-  }
+
+	utilruntime.Must(clientgoscheme.AddToScheme(mgr.GetScheme()))
+
+	utilruntime.Must(routev1.AddToScheme(mgr.GetScheme()))
+	// +kubebuilder:scaffold:scheme
```

Comment:
@theishshah Can you please fix this indentation issue and double check this. It still hasn't been fixed from the past 2 reviews.
![image](https://user-images.githubusercontent.com/6660644/82590917-622e1700-9b53-11ea-9caa-79ddb5f014a2.png)